city_scrapers/spiders/wayne_local_emergency_planning.py

```diff
@@ -0,0 +1,99 @@
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil import parser
+
+
+class WayneLocalEmergencyPlanningSpider(CityScrapersSpider):
+    name = "wayne_local_emergency_planning"
+    agency = "Wayne County Local Emergency Planning Committee"
+    timezone = "America/Detroit"
+    allowed_domains = ["www.waynecounty.com"]
+    start_urls = ["https://www.waynecounty.com/departments/hsem/wayne-county-lepc.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        # the meeting dates I've seen all had the substring 'day, ' as in 'Wednesday, March 6, 2019'
+        meeting_dates = response.xpath('''//p[contains(text(),'day, ')]''')
+
+        for item in meeting_dates:
+            # clean off the paragraph tags from the parsed meeting date (item)
+            item = str(item.extract()).replace('<p>', '').replace('</p>', '')
+            # print('------------')
+            # print(item)
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        prefix = 'Wayne County LEPC Meeting - '
+        # create a unique title for each meeting by combining
```

Comment:
For regular meetings like this, it's actually preferable to not have unique titles since we have the details elsewhere, so "Local Emergency Planning Committee" is fine

Reply:
I'll make that change on my branch now.